inspirehep/modules/disambiguation/api.py

```diff
@@ -34,6 +34,7 @@
     get_all_publications,
 )
 from inspirehep.modules.disambiguation.core.ml.models import EthnicityEstimator
+from inspirehep.modules.disambiguation.core.ml.models import DistanceEstimator
```

Comment:
The `inspire-next` project does not follow `beard`'s convention for imports, here we prefer to group them by module and we sort them in alphabetical order:
```python
from inspirehep.modules.disambiguation.core.ml.models import (
    DistanceEstimator,
    EthnicityEstimator,
)
```